### PR TITLE
Fixed the Json

### PR DESCRIPTION
**Pull request by DEVSOG12**

### CODE REVIEW

File: SDG_api/main.json [hunk 1]
```diff
@@ -768,13 +768,13 @@
             "definition":"Indicator 2.C.1 is the indicator of food price anomalies. \n Food price anomalies can be measured using the domestic food price volatility index. \n Domestic food price volatility index measures the variation (volatility) in domestic food prices over time - this is measured as the weighted-average of a basket of commodities based on consumer or market prices. High values indicate a higher volatility (more variation) in food prices.",
             "goal":"By 2030 adopt measures to ensure the proper functioning of food commodity markets and their derivatives and facilitate timely access to market information, including on food reserves, in order to help limit extreme food price volatility.",
             "more_research":"Further data and research can be found at the Our World in Data entry on Food Prices."
-          },          
+          }          },
```
Comment (Devjay1) [inline SDG_api/main.json:772]: That seems to be the cause of the Probelm  ;)

**Review by Devjay1 — APPROVED** [review]
LGTM !!

File: SDG_api/main.json (hunk 1 above)
Comment (Devjosh1) [reply SDG_api/main.json:772]: Its okay btw

**Review by Devjosh1 — APPROVED** [review]
It's Okay 👍

**Comment by DEVSOG12** [comment]
Al Done

**Pull request merged**